editor/store/selectors.js

```diff
@@ -300,6 +300,44 @@ export function isEditedPostEmpty( state ) {
 	);
 }
 
+/**
+ * Returns true if the post can be autosaved, or false otherwise.
+ *
+ * @param  {Object}  state Global application state
+ * @return {boolean}       Whether the post can be autosaved
+ */
+export function isPostAutosavable( state ) {
+	// If the post is autosaving, it is not autosavable.
+	if ( state.isAutosaving ) {
+		return false;
+	}
+
+	// If we don't already have an autosave, the post is autosavable.
+	if ( ! hasAutosave( state ) ) {
+		return true;
+	}
+
+	const title = getEditedPostAttribute( state, 'title' );
+	const excerpt = getEditedPostExcerpt( state );
+	const content = getEditedPostContent( state );
+	const autosave = state.editor.present.autosave;
+
+	// If the title, excerpt or content has changed, the post is autosavable.
+	if (
+		( title !== autosave.title.raw ) ||
+		( excerpt !== autosave.excerpt.raw ) ||
+		( content !== autosave.content.raw )
+	) {
+		return true;
```

Comment:
Minor: This condition could be eliminated and replaced with a single `return` value:

```js
return (
	( title !== autosave.title.raw ) ||
	( excerpt !== autosave.excerpt.raw ) ||
	( content !== autosave.content.raw )
);
```

Reply:
nice! changed in https://github.com/WordPress/gutenberg/pull/6257/commits/7e8b6a23f528af3d984948aabf2a180230d5f567